src/layouts/dashboard/sidebar/nav-mini.tsx

```diff
@@ -56,17 +55,38 @@ const NavMini = ({ width }: { width: number }) => {
           {
             name: "Videos",
             href: "/video/consulting",
-            icon: (sx: SxProps) => <VideoLibrary sx={sx} />,
+            icon: (sx: SxProps) => (
+              <Icon
+                icon="material-symbols:video-library-rounded"
+                width="32"
+                height="32"
+                color="#BDBDBD"
+              />
+            ),
           },
           {
             name: "Tests",
             href: "/tests",
-            icon: (sx: SxProps) => <Quiz sx={sx} />,
+            icon: (sx: SxProps) => (
+              <Icon
+                icon="healthicons:i-exam-multiple-choice"
+                width="32"
+                height="32"
+                color="#BDBDBD"
+              />
+            ),
           },
           {
             name: "Resources",
             href: "/resources",
-            icon: (sx: SxProps) => <FilePresent sx={sx} />,
+            icon: (sx: SxProps) => (
+              <Icon
```

Comment:
if all of these icons have the same width, height, and color could you create a component above this and take in the icon as a prop (i know this is probably my code lol)

Reply:
think has been changed since pulling down the latest changes